culture/code-of-conduct.md

```diff
@@ -0,0 +1,124 @@
+
+If we witness or receive a report about abusive behavior, we will contact the perpetrator to have a conversation with them and verify what has transpired, and they will be removed from the 2i2c community. Their Slack account will be deactivated, and permissions will be removed from any 2i2c-related repositories. They will not be welcome in any physical or digital spaces covered by the 2i2c Code of Conduct.
+
+[This is the protocol](response-protocol.md) that 2i2c Directors and Steering Council members will use to respond to reports of code of conduct violations.
+
+(coc:reporting)=
+## Reporting
+
+If you see a violation of our code of conduct, please [report it to the 2i2c Executive Director](https://docs.google.com/forms/d/e/1FAIpQLSeKFuoghGzftqqgaa0xllZyocdUJsWZmoSaFFlerimg4n_Y8A/viewform?usp=sf_link).
+
+
+### Why should I report?
+
+* **You are responsible for making 2i2c a safe and comfortable space for everyone.** Everyone in our community shares this responsibility. 2i2c Team and Steering Council members are not around the online spaces or at 2i2c events all the time, so we cannot enforce the code of conduct without your help.
+* **The consequences for the 2i2c community of not reporting bad behavior outweigh the consequences for one person of reporting it.** We sometimes hear “I don’t want X person to meet consequences because I told someone about their bad behavior.” Consider the impact on everyone else at 2i2c of letting their behavior continue unchecked.
+* 2i2c only works as an open, participatory, community-driven community because of shared trust between community members. **Reporting code of conduct violations helps us identify when this trust is broken, to prevent that from happening in the future.**
+
+
+### Where and how to report
+
+Please report all code of conduct violations using [our reporting form](https://docs.google.com/forms/d/e/1FAIpQLSeKFuoghGzftqqgaa0xllZyocdUJsWZmoSaFFlerimg4n_Y8A/viewform?usp=sf_link). This is a short Google Form that will be sent to the Executive Director.
+
+If you would rather discuss the matter in person with a team member, send a direct message in the 2i2c Slack, or email a Steering Council member to schedule a time to talk.
+
+
+### Confidentiality
+
+We will keep all reports confidential, except if we've discussed with you and agreed otherwise. When we discuss incidents with people who are reported, we will anonymize details as much as we can to protect reporter privacy.
+
+However, some incidents happen in one-on-one interactions, and even if the details are anonymized, the reported person may be able to guess who made the report. If you have concerns about retaliation or your personal safety, and do not want us to share the details of your report with anyone (including the perpetrator) please let us know explicitly in your report. Unfortunately, in that situation we will not be able to take any action.
```

Comment:
Hmm, does the recurse center coc say that they won't be able to take any action? how do other communities deal with this? This seems difficult but very important.

Reply:
Yeah this one is from their document. The one place where I have seen this not be true is with Title IX stuff in universities, but not sure if that applies to us. I think the question is whether there are some cases where we would take action even if the person explicitly asked us not to.